src/article/domain/repositories/user.repository.ts

```diff
@@ -1,5 +1,5 @@
 import { User } from '../entities/user.domain-entity';
 
```

Comment:
In my last [PR](https://github.com/jousmo/nestpoc-blog/pull/2/files#diff-9601a8f6c734c2001be34a2361f76946d19a39a709b5e8c624a2a5a0aade05f2R38) you accepted a new eslint rule, this rule validates naming convention for interfaces.